Assets/HoloToolkit-Examples/SpatialSound/Scripts/UAudioManagerTest.cs

```diff
@@ -17,13 +17,17 @@ private IEnumerator ContinouslyPlaySounds()
         {
             while (true)
             {
-                UAudioManager.Instance.PlayEvent("Laser");
+                UAudioManager.Instance.PlayEvent("Vocals3d");
```

Comment:
Is there a way we can make this not based on strings?

Reply:
Every event needs a unique ID and to support audio event banks in downloadable asset bundles I'd like to avoid any build time dependencies (i.e. generating an ID table).  There are several ideas kicking around in my head on how to do this, but each one requires some more thought / discussion.
For a pure unity editor experience, we could provide an editor attribute that would convert (in this case) a string field into a dropdown to select an event from, but this still doesn't help with the potential of event renaming and the connection being "lost".
Perhaps this is one for a future update after more thought.

Reply:
You might be able to do something similar with how we handle voice keywords and commands in the main toolkit, with a serialized array.

Reply:
I've added a custom property drawer to alleviate having to enter string values manually.